src/disco/reducers/discoResults.js

```diff
@@ -1,8 +1,13 @@
-import { DISCO_RESULTS } from 'disco/constants';
+import { LOAD_DISCO_RESULTS } from 'disco/constants';
 
 export default function discoResults(state = [], { type, payload }) {
-  if (type === DISCO_RESULTS) {
-    return payload.results;
+  switch (type) {
+    case LOAD_DISCO_RESULTS: {
+      const { entities, result } = payload;
+      // Unwrap the magical discoResults API schema.
```

Comment:
Your comment here makes more sense than the comment 😄

Could you add a note saying the API schema that transforms `result.results` can be found in the `disco/api.js` file? Until #2917 is fixed it's probably best to over-comment that.